invisible_cities/cities/esmeralda_test.py

```diff
@@ -199,3 +200,22 @@ def test_esmeralda_filters_events_with_too_many_hits(esmeralda_config, Th228_tra
     assert (summary.evt_ntrks > 0        ).tolist() == evt_pass
     assert (summary.evt_nhits < nhits_max).tolist() == evt_pass
     assert            filter_output.passed.tolist() == evt_pass
+
+
+def test_esmeralda_does_not_crash_with_no_hits(esmeralda_config, Th228_hits, config_tmpdir):
+    path_in   = os.path.join(config_tmpdir, "esmeralda_does_not_crash_with_no_hits_input.h5")
+    path_out  = os.path.join(config_tmpdir, "esmeralda_does_not_crash_with_no_hits_output.h5")
+    esmeralda_config.update(dict( files_in    = path_in
+                                , file_out    = path_out
+                                , event_range = 1))
+
+    # copy file and remove the hits from the first event
+    shutil.copy(Th228_hits, path_in)
+    with tb.open_file(path_in, "r+") as file:
+        first_evt = file.root.Run.events[0][0]
+        evt_rows  = [row[0] == first_evt for row in file.root.RECO.Events]
+        n_delete  = sum(evt_rows)
+        file.root.RECO.Events.remove_rows(0, n_delete)
+
+    # just test that it doesn't crash
+    esmeralda(**esmeralda_config)
```

Comment:
I'm assuming that Beersheba works identically to this, and so it doesn't need a test to verify it doesn't crash?